extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -1258,10 +1258,21 @@ export class Bind {
   reportWorkerError_(e, message, opt_element) {
     const userError = user().createError(message + ' ' + e.message);
     userError.stack = e.stack;
-    reportError(userError, opt_element);
+    this.reportError_(userError, opt_element);
     return userError;
   }
 
+  /**
+   * @param {!Error} error
+   * @param {!Element=} opt_element
+   */
+  reportError_(error, opt_element) {
```

Comment:
I'm puzzled about why this is required, since we already stub `reportError` during tests. 

I can investigate separately.